src/main/java/com/redhat/sast/api/model/Job.java

```diff
@@ -11,7 +11,9 @@
 import lombok.NoArgsConstructor;
 
 @Entity
-@Table(name = "job")
+@Table(
+        name = "job",
+        indexes = {@Index(name = "idx_job_id", columnList = "id")})
```

Comment:
This is the default behaviour. Because we have decorated the `id` property with `@Id`, which will be the primary key for the schema, therefore, an autogenerated index will be created anyway for the primary.